domains/certificates/Query.API/API.UnitTests/MeasurementsSyncer/SlidingWindowServiceTest.cs

```diff
@@ -19,6 +20,8 @@ public class SlidingWindowServiceTest
     private readonly Gsrn _gsrn = new Gsrn(GsrnHelper.GenerateRandom());
     private readonly UnixTimestamp _now = UnixTimestamp.Now();
     private readonly IMeasurementSyncMetrics _measurementSyncMetrics = Substitute.For<IMeasurementSyncMetrics>();
+    private readonly MeasurementsSyncOptions _options = Substitute.For<MeasurementsSyncOptions>();
+
 
     public SlidingWindowServiceTest()
```

Comment:
Add tests of how the sliding window is expected to look like with the new age requirement.